src/fu-history.c

```diff
@@ -1331,12 +1383,13 @@ fu_history_add_security_attribute(FuHistory *self,
 	/* remove entries */
 	locker = g_rw_lock_writer_locker_new(&self->db_mutex);
 	g_return_val_if_fail(locker != NULL, FALSE);
-	rc = sqlite3_prepare_v2(self->db,
-				"INSERT INTO hsi_history (hsi_details, hsi_score)"
-				"VALUES (?1, ?2)",
-				-1,
-				&stmt,
-				NULL);
+	rc = sqlite3_prepare_v2(
+	    self->db,
+	    "INSERT INTO hsi_history (hsi_details, hsi_score, fwupd_version, changes)"
```

Comment:
doesn't changing this mean a database migration?

Reply:
I modified the table schema and try to move the data to the new schema.